services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/kafka/RecordHandler.java

```diff
@@ -633,8 +634,8 @@ private void doDumpRulesRequest(final CommandMessage message,  String replyToTop
             context.getKafkaProducer().postMessage(replyToTopic, infoMessage);
         } catch (SwitchOperationException e) {
             logger.info("Dump rules is unsuccessful. Switch {} not found", request.getSwitchId());
-            ErrorData errorData = new ErrorData(ErrorType.DATA_INVALID, e.getMessage(),
-                    request.getSwitchId().toString());
+            ErrorData errorData = new DumpRulesErrorData(ErrorType.NOT_FOUND, e.getMessage(),
+                    "The switch was not found when requesting a rules dump.");
```

Comment:
It would be great to add a switch id to the error message.

Reply:
This is the error description. The error message contains the switch id.